prepare-build.sh

```diff
@@ -17,11 +17,11 @@
 declare -a devices=("anthias" "bass" "beluga" "catfish" "dory" "firefish" "harmony" "hoki" "koi" "inharmony" "lenok" "minnow" "mooneye" "narwhal" "nemo" "pike" "qemux86" "ray" "rinato" "smelt" "sparrow" "sparrow-mainline" "sprat" "sturgeon" "sawfish" "skipjack" "swift" "tetra" "triggerfish" "wren")
 
 declare -a layers=(
-    "src/oe-core                   https://github.com/openembedded/openembedded-core.git scarthgap"
-    "src/oe-core/bitbake           https://github.com/openembedded/bitbake.git           2.8.1"
-    "src/meta-openembedded         https://github.com/openembedded/meta-openembedded.git scarthgap"
-    "src/meta-qt5                  https://github.com/meta-qt5/meta-qt5                  scarthgap"
-    "src/meta-smartphone           https://github.com/shr-distribution/meta-smartphone   scarthgap"
+    "src/oe-core                   https://github.com/openembedded/openembedded-core.git walnascar"
+    "src/oe-core/bitbake           https://github.com/openembedded/bitbake.git           2.12"
+    "src/meta-openembedded         https://github.com/openembedded/meta-openembedded.git walnascar"
+    "src/meta-qt5                  https://github.com/meta-qt5/meta-qt5                  master 4d3ebeb76f2296e31598125742bf4899a21bfc30"
```

Comment:
I used the `walnascar` branch of meta-qt5 and it worked just fine.

Reply:
Good suggestion. I think I used the fixed commit as at the time there wasn't a walnascar branch. Will fix later.